### PR TITLE
Update neurio_energy.markdown

### PR DESCRIPTION
**Pull request by ckaczor**
- Update Homepage URL and Callback URL as they are currently required but do not have to be real URLs
- Update YAML structure
- Indicate that sensor_id has to be quoted so it doesn't get converted to a number when used in code

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @ckaczor, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#18000).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/neurio_energy.markdown [hunk 1]
```diff
@@ -11,17 +11,17 @@ ha_platforms:
...
 ```yaml
 # Example configuration.yaml entry
 sensor:
-  platform: neurio_energy
-  api_key: CLIENT_ID
-  api_secret: CLIENT_SECRET
-  sensor_id: SENSOR_ID
+  - platform: neurio_energy
+    api_key: CLIENT_ID
+    api_secret: CLIENT_SECRET
```
Comment (frenck) [inline source/_integrations/neurio_energy.markdown]:
```suggestion
    api_key: "CLIENT_ID"
    api_secret: "CLIENT_SECRET"
```

File: source/_integrations/neurio_energy.markdown [hunk 2]
```diff
@@ -11,17 +11,17 @@ ha_platforms:
   - sensor
 ---
 
-Integrate your [Neurio](https://neur.io/) meter information into Home Assistant. To get an API key and secret, login to your [Neurio account](https://my.neur.io/#settings/applications/register) and register an application. Note the Homepage URL and Callback URL are optional.
+Integrate your [Neurio](https://neur.io/) meter information into Home Assistant. To get an API key and secret, login to your [Neurio account](https://my.neur.io/#settings/applications/register) and register an application. Note the Homepage URL and Callback URL are required but can be "fake" URLs.
```
Comment (frenck) [inline source/_integrations/neurio_energy.markdown]: Let's remove the requisites out of the introduction and make it part and make it clear part of the configuration instructions.

File: source/_integrations/neurio_energy.markdown [hunk 3]
```diff
@@ -39,7 +39,7 @@ api_secret:
   required: true
   type: string
 sensor_id:
-  description: "The sensor ID, a hex number as shown on the [PWRView webpage](https://mypwrview.generac.com/#settings/sensors), e.g., `0x0000XXXXXXXXXXXX`."
+  description: "The sensor ID, a hex number as shown on the [PWRView webpage](https://mypwrview.generac.com/#settings/sensors), e.g., `0x0000XXXXXXXXXXXX`.  The value must be in double quotes so it is kept a string instead of being interpreted as a number."
```
Comment (frenck) [inline source/_integrations/neurio_energy.markdown]: This is not specific to this integration, but part of YAML.
We don't report this with every field we have. Please revert this part.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/neurio_energy.markdown (hunk 2 above)
Comment (ckaczor) [reply source/_integrations/neurio_energy.markdown]: I removed the sentence about the required fields but I'm not sure what you mean for the second part.  Create a new paragraph describing how to setup the Neurio application in their account settings?

Comment (frenck) [reply source/_integrations/neurio_energy.markdown]: The general structure of an integration page is:

- Introduction about the integration and/or products
- Optionally a chapter about prerequisites to use it (e.g., set up devices, tokens and such)
- A chapter about configuring the integration.

This structure is missing, hence now the introduction is kinda minimal and mixed with prerequisites.

**Pull request closed without merging**